File: workplace-search-client/src/main/java/com/seanjstory/workplace/search/client/ssl/NullHostnameVerifier.java
package com.seanjstory.workplace.search.client.ssl;

import javax.net.ssl.HostnameVerifier;
import javax.net.ssl.SSLSession;

/**
 * A HostnameVerifier that verifies all hosts
 */
public class NullHostnameVerifier implements HostnameVerifier {

    @Override
    public boolean verify(String hostname, SSLSession session) {
        return true;
    }
}

File: workplace-search-client/src/main/java/com/sstory/workplace/search/client/ssl/NullHostnameVerifier.kt
package com.sstory.workplace.search.client.ssl

import javax.net.ssl.HostnameVerifier
import javax.net.ssl.SSLSession

internal class NullHostnameVerifier : HostnameVerifier {
    override fun verify(hostname: String, session: SSLSession): Boolean {
        return true
    }
}
